commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -41,13 +36,31 @@
       arity = "1")
   private boolean slashingProtectionEnabled = false;
```

Comment:
think slashing needs to be enabled by default - do we want to raise a ticket to change this over?

Reply:
yes I think we do. but after it's implemented. so think raise another to enable it by default once we are done with all the pieces.